### PR TITLE
Heder

### PR DESCRIPTION
**Pull request by Alba-Lucia**
Header

### CODE REVIEW

File: components/Hero/Hero.js [hunk 1]
```diff
@@ -1,14 +1,25 @@
...
+import { Wave } from './Wave'
 
 export function Hero() {
   return (
-    
-    <div className="hero float-end m-4 w-100">
-      <div className="hero__content pe-4 text-end">
-        <p>Hello I{`'`}m</p>
-        <h1>Abla Lucia Gonzalez</h1>
+   <div className="content__hero--blue mb-4">
```
Comment (Krank3n) [inline components/Hero/Hero.js:6]: can we change the classes where the BEM Block starts with the component name:

hero__content--blue

Comment (Alba-Lucia) [reply components/Hero/Hero.js:6]: Ok sure

File: components/Hero/Wave.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            style={{height: '150px', overflow: 'hidden'}} >
+              <svg viewBox="0 0 500 150" preserveAspectRatio="none" 
+              style={{height: '100%', width: '100%'}}>
+                <path d="M-9.73,3.46 C185.75,75.49 322.83,81.41 511.02,1.49 L501.29,-16.27 L-1.62,-16.27 Z" 
+                  style={{stroke: 'none', fill: '#132426'}}>
+                </path>
+              </svg>
+          </div>
+          </div>  
+    </div>
```
Comment (Krank3n) [inline components/Hero/Wave.js:30]: `    <div className='wave__content--desktop'>
        <div className="wave__hero">
          <div className="wave__hero">
              <svg viewBox="0 0 500 150" preserveAspectRatio="none">
                <path d="M-4.05,-2.46 C288.77,292.59 244.16,-148.51 502.91,84.38 L506.67,-11.34 L-1.62,-16.27 Z">
                </path>
              </svg>
          </div>
        </div>

          <div className="wave__hero--mobile"> 
           <div className="wave__hero">
              <svg viewBox="0 0 500 150" preserveAspectRatio="none">
                <path d="M-9.73,3.46 C185.75,75.49 322.83,81.41 511.02,1.49 L501.29,-16.27 L-1.62,-16.27 Z">
                </path>
              </svg>
          </div>
          </div>  
    </div>`

Comment (Krank3n) [reply components/Hero/Wave.js:30]: can we please remove the inline styles and add it to wave.css file ?

Comment (Krank3n) [reply components/Hero/Wave.js:30]: Can we move the Wave component into it's own folder.

Wave/wave.js
Wave/wave.css

File: components/Hero/Hero.js (hunk 1 above)
Comment (Alba-Lucia) [reply components/Hero/Hero.js:6]: It's ready

File: styles/globals.css [hunk 3]
```diff
@@ -1,19 +1,63 @@
 @import '../components/Hero/hero.css';
```
Comment (Krank3n) [inline styles/globals.css:1]: Can we use SCSS?

image

https://nextjs.org/docs/basic-features/built-in-css-support

File: pages/index.js [hunk 4]
```diff
@@ -4,12 +4,15 @@ import Image from 'next/image'
...
+import { Wave } from '../components/Hero/Wave'
+
 
 export default function Home() {
   return (
     <div>
-      <div className="container m-4">
-        <Hero />
+      
+      <div className="">
```
Comment (Krank3n) [inline pages/index.js:14]: maybe remove className if you are not using it :)

File: components/Hero/wave.module.css [hunk 5]
```diff
@@ -0,0 +1,6 @@
+.header__wave-desktop{
+    background-color: red;
+    margin-top: 90px;
+
```
Comment (Krank3n) [inline components/Hero/wave.module.css:4]: remove empty space

File: components/Hero/Hero.js [hunk 6]
```diff
@@ -1,14 +1,25 @@
...
+        <div className="container d-flex flex-column flex-md-row justify-content-between align-items-center">
+          <div className=" order-md-1 ">
+            <div className="hero my-4">
+              <div className=" pe-4 text-end w-100">
+                <p>Hello I{`'`}m</p>
+                <h1>Alba Lucía González</h1>
+              </div>
+            </div>
+          </div>
+          <img src='AlbaPortafolio.png' alt='Photo alba lucia gonzalez' className="hero__imagen"/>
```
Comment (Krank3n) [inline components/Hero/Hero.js:17]: are we using english or spanish ??? Maybe best to practice in English, because I know you can ;)

hero__image

Comment (Krank3n) [inline components/Hero/Hero.js:17]: `Photo Alba Lucia Gonzalez`

File: components/Hero/hero.css [hunk 7]
```diff
@@ -1,3 +1,40 @@
...
+}
+
+.wave{
+    margin-top: -3px;
+}
+
+.header__wave--desktop{
+    display: none;
+}
+@media (min-width: 768px) {
```
Comment (Krank3n) [inline components/Hero/hero.css:33]: can you move this into wave.css file?

File: components/Hero/hero.css [hunk 8]
```diff
@@ -1,3 +1,40 @@
...
+    z-index: 9;
+}
+@media (min-width: 768px){
+    .hero__imagen{
+        width: 50%;
+        margin-bottom: -10rem;
+    }
+}
+@media (min-width: 1024px){
+    .hero__imagen{
```
Comment (Krank3n) [inline components/Hero/hero.css:20]: .hero__image

when you use SCSS it will be:

**Review by Krank3n — CHANGES_REQUESTED** [review]
Beautiful start :) Really impressed with your skills so far

File: components/Hero/Wave.js (hunk 2 above)
Comment (Alba-Lucia) [reply components/Hero/Wave.js:30]: can we please remove the inline styles and add it to wave.css file ?

- Which line?

File: styles/globals.css (hunk 3 above)
Comment (Alba-Lucia) [reply styles/globals.css:1]: I try work install scss whith storybook but no was possible I don how work

File: components/Hero/Hero.js (hunk 6 above)
Comment (Alba-Lucia) [reply components/Hero/Hero.js:17]: oops! I forgot used English

File: styles/globals.css (hunk 3 above)
Comment (Krank3n) [reply styles/globals.css:1]: it's not associated with storybook

just do 'npm install --save-dev sass' on your main project folder

**Pull request merged**